sources/safe/zkevm/safe_zkevm_sources.yml

```
@@ -0,0 +1,45 @@
+version: 2
+
+sources:
+  - name: gnosis_safe_zkevm
+    freshness:
+      warn_after: { count: 12, period: hour }
+    tables:
+      - name: GnosisSafeProxyFactory_v1_3_0_evt_ProxyCreation
```

I think here may be a typo.
```suggestion
      - name: GnosisSafeProxyFactory_v_1_3_0_evt_ProxyCreation
```